src/cocaine-app/db/mongo/__init__.py

```diff
@@ -18,7 +20,7 @@ def save(self):
             logger.debug('Object with id {0} has no _dirty flag set'.format(self.id))
             return
 
-        res = self.collection.update({'id': self.id}, self.dump(), upsert=True)
+        res = self.collection.update({self.PRIMARY_ID_KEY: self.id}, self.dump(), upsert=True)
```

Comment:
Is it possible to use kwargs for `{self.PRIMARY_ID_KEY: self.id}` and `self.dump()`?